### DISCUSSION TITLE
Using options (e.g. defaultOptions/configure) with insertContentAt

### DISCUSSION THREAD

**Discussion by jloganolson** [ueberdosis/tiptap#1702] [General Questions]
How can I set an option when creating a node with insertContentAt - the following does not worK:
```
          commands.insertContentAt({ from: prePos, to: postPos }, {
            type: 'chip',
            options:{
              chipContent: "test"
            }
          })
```

**Comment by philippkuehn** [comment]
do you mean attributes? try this:

```js
commands.insertContentAt({ from: prePos, to: postPos }, {
  type: 'chip',
  attrs:{
    chipContent: "test"
  }
})
```

**Reply by jloganolson** [reply]
I was using options (since I didn't want it to be an html attribute - just text content), e.g. 
```
export default Node.create({
  name: 'chip',

  defaultOptions: {
    chipContent: "stan",
  },
```
then later down in renderhtml:
```  
renderHTML({ HTMLAttributes }) {
    return ['span', mergeAttributes(HTMLAttributes), this.options.chipContent]
  },
```

Am I misusing that feature (i.e. should I be doing this with addAttributes)?

**Reply by hanspagel** [reply]
Options are the same for all nodes of a type, attributes are attached to the specific node. Yes, you want to use addAttributes 👌
